### PR TITLE
feat: enable ledger

### LINKED ISSUE #131: Enable Ledger in cosmos-kit
Should be easy enough to add https://docs.cosmology.zone/cosmos-kit/integrating-wallets/ledger

### PR DESCRIPTION
**Pull request by fmorency**
Enable the ledger wallet from Cosmos-Kit.

I think we need `https` to test this.

Fixes #131 

## Summary by CodeRabbit

- **New Features**
	- Expanded wallet selection in the browser section to include 'Ledger' wallets.
	- Added a "Contacts" button in the MobileNav for improved user interaction.
	- Enhanced flexibility in chain configuration across multiple components by utilizing environment variables.

- **Bug Fixes**
	- Updated endpoint configuration to dynamically reference the environment variable for improved adaptability.

- **Chores**
	- Removed unused import from the Admins component.
	- Updated dependency versions in package.json for better performance and features.
	- Removed the `isValidAddress` function from utilities.

### CODE REVIEW

**Comment by fmorency** [comment]
@chalabi2 I'm unable to make it work, even when using https. I always get 



Ideas?

**Review by chalabi2 — DISMISSED** [review]
Lgtm! Will add anything that's missing if it doesn't work with https as per a signal message you left

**Comment by fmorency** [comment]
@chalabi2 are you able to make this work?

**Comment by chalabi2** [comment]
Are you properly installing the wallet package and importing

`import { wallets as ledger } from "@cosmos-kit/ledger";`

Then adding to 

`combinedWallets = [...wallets, ...web3authwallets, ...ledger]`

Also please consider the cosmology docs as they warn users


```
Browser Support

This package uses the [WebUSB](https://caniuse.com/webusb)

API to connect to Ledger devices.
We recommend using the latest version of Chrome and Chrome Android.

https://developer.mozilla.org/en-US/docs/Web/API/USB/getDevices

https://developer.mozilla.org/en-US/docs/Web/API/USB/requestDevice

```

**Comment by chalabi2** [comment]
Found the issue. 

The ledger wallet has a util file 

https://github.com/cosmology-tech/cosmos-kit/blob/14fbaa738a014a72668fc59b24c7f6848dcaffa7/wallets/ledger/src/web-usb-hid/utils.ts#L22

This specific function imports the chains from the chain registry and sifts through them until it finds your chains chain id. Then it grabs your chains bech 32 prefix. 

Which if it cant find errors here

https://github.com/cosmology-tech/cosmos-kit/blob/14fbaa738a014a72668fc59b24c7f6848dcaffa7/wallets/ledger/src/web-usb-hid/client.ts#L32

Tested it by manually setting prefix to manifest and building the ledger adapters then linking locally + using localCan for local https and it worked. 

Moving forward if anything from cosmos kit is failing silently you can set the 

[logLevel](https://github.com/liftedinit/manifest-app/blob/5293618a0aad648847a17311ff131b574c928bfe/pages/_app.tsx#L177) in _app.tsx to error or debug for more info.

**Review by chalabi2 — CHANGES_REQUESTED** [review]
Found the issue.

The ledger wallet has a util file

https://github.com/cosmology-tech/cosmos-kit/blob/14fbaa738a014a72668fc59b24c7f6848dcaffa7/wallets/ledger/src/web-usb-hid/utils.ts#L22

This specific function imports the chains from the chain registry and sifts through them until it finds your chains chain id. Then it grabs your chains bech 32 prefix.

Which if it cant find errors here

https://github.com/cosmology-tech/cosmos-kit/blob/14fbaa738a014a72668fc59b24c7f6848dcaffa7/wallets/ledger/src/web-usb-hid/client.ts#L32

Tested it by manually setting prefix to manifest and building the ledger adapters then linking locally + using localCan for local https and it worked.

Moving forward if anything from cosmos kit is failing silently you can set the

[logLevel](https://github.com/liftedinit/manifest-app/blob/5293618a0aad648847a17311ff131b574c928bfe/pages/_app.tsx#L177) in _app.tsx to error or debug for more info.

**Comment by fmorency** [comment]
Oh, I see, thanks! We should update the chain-registry before merging this PR then?

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "**/@cosmjs/stargate": "npm:@liftedinit/stargate@0.32.4-ll.3"
   },
   "dependencies": {
-    "@chain-registry/assets": "^1.64.83",
+    "@chain-registry/assets": "^1.70.65",
```
[the remark this answers is not in the document]
Comment (fmorency) [reply package.json:33]: It's already contained within the react package

**Comment by fmorency** [comment]
This feature works on the Testnet since https://github.com/cosmos/chain-registry/pull/5719 was merged and `chain-registry` bumped.

This feature will not work on QA and Mainnet until we add those networks to the chain-registry.

**Pull request merged**